executor/ddl.go

```diff
@@ -152,6 +154,36 @@ func (e *DDLExec) executeCreateTable(s *ast.CreateTableStmt) error {
 	return errors.Trace(err)
 }
 
+func (e *DDLExec) executeCreateView(s *ast.CreateViewStmt) error {
+	ident := ast.Ident{Schema: s.ViewName.Schema, Name: s.ViewName.Name}
+
+	// SelectFields represents the field expression or field AsName in SelectStmt
+	// and saves it as a string to establish the mapping relationship of ColList
+	var selectFields []string
+	compiler := Compiler{}
+	stmt, err := compiler.Compile(e.ctx, s.Select)
```

Comment:
The `Compile` do a complete process of optimization, but what you need is only a schema.
You have built logical plan in planbuilder. Why not save the schema of select in DDL plan?